package.json

```diff
@@ -33,7 +33,7 @@
     "mkdirp": "^0.5.1",
     "mongodb": "3.3.0",
     "mysql": "^2.11.1",
-    "mysql2": "1.4.2",
+    "mysql2": "^1.7.0",
```

Comment:
We tend to set strict dependency version to reduce the uncertainty of the install dependency versions.
1. Using `^` should not be a problem if the dependencies follow the major/minor/patch rules but we cannot be sure of that.
2. I am not a fan of the idea that, depending of the date you install a package, it dependencies could change. (especially for debug purpose it could be painful)